Innovative Flix/force-app/main/default/objects/Episode__c/validationRules/Number_Of_Season_Greather_Than_0.validationRule-meta.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Number_Of_Season_Greather_Than_0</fullName>
```

Comment:
Is this name correct? Shouldn't be "episode number"?

Reply:
Done